Bottom: acb3e62c240f344d112f8a0064b6d5be76dda8a1
Top:    f3f0fb6da3f912c338b2ab22d8ec52ff537daced
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2008-11-04 09:45:31 -0500

Refresh of jds-lirc

---

diff --git a/drivers/input/ir/Makefile b/drivers/input/ir/Makefile
index 0bdafb0..2ccdda3 100644
--- a/drivers/input/ir/Makefile
+++ b/drivers/input/ir/Makefile
@@ -6,7 +6,3 @@
 obj-$(CONFIG_INPUT_IR)		+= ir.o
 ir-objs := ir-core.o ir-configfs.o
 
-
-obj-$(CONFIG_IR_GPT)		+= ir-gpt.o
-obj-$(CONFIG_IR_MCEUSB2)	+= ir-mceusb2.o
-
